### PR TITLE
mode change, ads quick start

### PR DESCRIPTION
**Pull request by lec-bit**

### CODE REVIEW

File: content/en/docs/setup/quickstart.md [hunk 1]
```diff
@@ -79,6 +79,29 @@ time="2024-04-25T13:17:41Z" level=info msg="cni config file: /etc/cni/net.d/10-k
 time="2024-04-25T13:17:41Z" level=info msg="command Start cni successful" subsys=manager
 ```
 
+### Mode change
+We provide two modes, ads and workload, for users to choose from. The default mode is currently workload. If we wishes to use the ads mode,  only need to modify a single startup parameter.
```
Comment (hzxuzhonghu) [inline content/en/docs/setup/quickstart.md:83]:
```suggestion
We provide two modes: `ads` and `workload`, for users to choose from. The default mode is currently workload. If we wish to use the ads mode,  only need to modify a single startup parameter `--mode`.
```

File: content/en/docs/setup/quickstart.md [hunk 2]
```diff
@@ -79,6 +79,29 @@ time="2024-04-25T13:17:41Z" level=info msg="cni config file: /etc/cni/net.d/10-k
...
+# change "--mode=workload" into "--mode=ads"
+helm upgrade kmesh ./deploy/helm -n kmesh-system
+
+# Install from Yaml
+vim deploy/yaml/kmesh.yaml
+...
+args: ["./start_kmesh.sh --mode=workload"]
+...
+# change "--mode=workload" into "--mode=ads"
+kubectl apply -f ./deploy/yaml/
```
Comment (hzxuzhonghu) [inline content/en/docs/setup/quickstart.md:100]: Long term we may stop maintaining the static yamls, so prefer helm install --set deploy.kmesh.containers.kmeshDaemonArgs=

**Review by hzxuzhonghu — COMMENTED** [review]
For ads mode, we should explictly document the requirements

File: content/en/docs/setup/quickstart.md [hunk 3]
```diff
@@ -79,6 +79,29 @@ time="2024-04-25T13:17:41Z" level=info msg="cni config file: /etc/cni/net.d/10-k
 time="2024-04-25T13:17:41Z" level=info msg="command Start cni successful" subsys=manager
 ```
 
+### Mode change
+We provide two modes, ads and workload, for users to choose from. The default mode is currently workload. If we wishes to use the ads mode,  only need to modify a single startup parameter.
+
+```shell
+# Install from Helm
```
Comment (nlgwcy) [inline content/en/docs/setup/quickstart.md:86]: separate into two parts.

File: content/en/docs/setup/quickstart.md [hunk 4]
```diff
@@ -79,6 +79,29 @@ time="2024-04-25T13:17:41Z" level=info msg="cni config file: /etc/cni/net.d/10-k
...
+vim deploy/helm/valuse.yaml
```
Comment (nlgwcy) [inline content/en/docs/setup/quickstart.md:87]:
```suggestion
vim deploy/helm/value.yaml
```

File: content/en/docs/setup/quickstart.md [hunk 5]
```diff
@@ -20,7 +20,7 @@ Kmesh needs to run on a Kubernetes cluster. Kubernetes 1.26, 1.27, 1.28 are curr
...
 
-Currently, Kmesh connects to the Istio control plane. Before starting Kmesh, install the Istio control plane software. We commend to install istio ambient mode because Kmesh ads mode need it. For details, see [ambient mode istio](https://istio.io/latest/docs/ops/ambient/getting-started/).
+Currently, Kmesh connects to the Istio control plane. Before starting Kmesh, install the Istio control plane software. We commend to install istio ambient mode because Kmesh workload mode need it. For details, see [ambient mode istio](https://istio.io/latest/docs/ops/ambient/getting-started/).
```
Comment (hzxuzhonghu) [inline content/en/docs/setup/quickstart.md:23]: commend/recommend

File: content/en/docs/setup/quickstart.md [hunk 6]
```diff
@@ -79,6 +79,30 @@ time="2024-04-25T13:17:41Z" level=info msg="cni config file: /etc/cni/net.d/10-k
...
+We provide two modes: `ads` and `workload`, for users to choose from. The default mode is currently workload. If we wish to use the ads mode,  only need to modify a single startup parameter `--mode`.
+
+```shell
+# Install from Helm
+vim deploy/helm/value.yaml
+...
+kmeshDaemonArgs: "--mode=workload --enable-bypass=false --enable-bpf-log=true"
+...
+# change "--mode=workload" into "--mode=ads"
+helm upgrade kmesh ./deploy/helm -n kmesh-system
```
Comment (hzxuzhonghu) [inline content/en/docs/setup/quickstart.md:92]: helm install

Comment (hzxuzhonghu) [reply content/en/docs/setup/quickstart.md:92]: I rthink you have installed kmesh before, so donot need --create-namespace

File: content/en/docs/userguide/route.md
Comment (hzxuzhonghu) [inline content/en/docs/userguide/route.md:1]: why add this file again, donot we have https://kmesh.net/en/docs/userguide/tcp_authorization/

Comment (lec-bit) [reply content/en/docs/userguide/route.md:1]: sorry, i mistake branch

**Pull request closed without merging**